pyproject.toml

```
@@ -0,0 +1,9 @@
+[tool.black]
+target-version = ['py311']
```

```suggestion
[tool.black]
target-version = ['py311']
line-length = 100
```

Can I request for us to use a line length of 100 characters?
Some context: https://knox.codes/posts/line-length-limits